cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -1904,6 +1910,26 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    ig.push(
+        Inst::new(
+            "iadd_ifcin",
+            r#"
+        Add integers with carry in.
+
+        Same as `iadd` with an additional carry input. Computes:
```

Comment:
nit: for all the comments of these instructions, can you add "flags" before "input" or "output", as needed, as you did for subtracts, please?